### PR TITLE
Update the Chinese translation of kubectl

### PR DESCRIPTION
**Pull request by hwdef**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Optimize some existing Chinese translations and add some Chinese translations that are not yet available

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
Optimization of kubectl Chinese translation
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 1]
```diff
@@ -124,21 +124,20 @@ msgid ""
 "specified or\n"
 "\t\tnot supported, convert to latest version.\n"
 "\n"
-"\t\tThe default output will be printed to stdout in YAML format. One can use "
-"-o option\n"
+"\t\tThe default output will be printed to stdout in YAML format. One can use -"
+"o option\n"
```
[the remark this answers is not in the document]
Comment (hwdef) [reply staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:128]: I don't quite understand what you mean. If you mean a change of format. This format is automatically changed by `poedit` software, I did not actively change the format.

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 2]
```diff
@@ -674,51 +649,47 @@ msgstr "指定容器要运行的镜像."
...
-"认值"
+msgstr "容器的镜像拉取策略。 如果留空，该值将不由客户端指定，由服务器默认"
 
 # https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/create_pdb.go#L62
 #: staging/src/k8s.io/kubectl/pkg/cmd/create/create_pdb.go:111
-msgid ""
-"The minimum number or percentage of available pods this budget requires."
-msgstr "最小数量百分比可用的 pods 作为 budget 要求."
+msgid "The minimum number or percentage of available pods this budget requires."
+msgstr "此预算要求的可用 pod 的最小数量或百分比。"
```
Comment (tengqm) [inline staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:657]:
```suggestion
msgstr "此预算要求的可用 Pod 的最小数量或百分比。"
```

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 3]
```diff
@@ -674,51 +649,47 @@ msgstr "指定容器要运行的镜像."
...
-msgstr "名称为最新创建的对象."
+msgstr "新创建的对象的名称。"
 
 # https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/autoscale.go#L71
 #: staging/src/k8s.io/kubectl/pkg/cmd/autoscale/autoscale.go:125
 msgid ""
 "The name for the newly created object. If not specified, the name of the "
 "input resource will be used."
-msgstr "名称为最新创建的对象. 如果没有指定, 输入资源的 名称即将被使用."
+msgstr "新创建的对象的名称。 如果未指定，将使用输入资源的名称。"
```
Comment (tengqm) [inline staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:669]:
```suggestion
msgstr "新创建的对象的名称。如果未指定，将使用输入资源的名称。"
```

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 4]
```diff
@@ -674,51 +649,47 @@ msgstr "指定容器要运行的镜像."
...
-"and 'service/v2'. The only difference between them is that service port in "
-"v1 is named 'default', while it is left unnamed in v2. Default is 'service/"
-"v2'."
+"and 'service/v2'. The only difference between them is that service port in v1 "
+"is named 'default', while it is left unnamed in v2. Default is 'service/v2'."
 msgstr ""
-"使用 generator 的名称. 这里有 2 个 generators: 'service/v1' 和 'service/v2'. "
-"为一个不同地方是服务端口在 v1 的情况下叫 'default', 如果在 v2 中没有指定名"
-"称. 默认的名称是 'service/v2'."
+"要使用的API生成器的名称。有两个生成器。service/v1' 和 'service/v2'。它们之间唯"
```
Comment (tengqm) [inline staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:678]:
```suggestion
"要使用的 API 生成器的名称。有两个生成器。'service/v1' 和 'service/v2'。它们之间唯"
```

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 5]
```diff
@@ -674,51 +649,47 @@ msgstr "指定容器要运行的镜像."
...
+"一的区别是，v1 中的服务端口被命名为 'default'，如果在 v2 中没有指定名称。默认"
+"是 'service/v2'。"
 
 # https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/expose.go#L99
 #: staging/src/k8s.io/kubectl/pkg/cmd/expose/expose.go:148
 msgid "The network protocol for the service to be created. Default is 'TCP'."
-msgstr "创建 service 的时候伴随着一个网络协议被创建. 默认是 'TCP'."
+msgstr "要创建的服务的网络协议。 默认为“TCP”。"
```
Comment (tengqm) [inline staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:685]:
```suggestion
msgstr "要创建的服务的网络协议。默认为“TCP”。"
```

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 6]
```diff
@@ -727,34 +698,33 @@ msgid ""
 "memory=512Mi'.  Note that server side components may assign limits depending "
 "on the server configuration, such as limit ranges."
 msgstr ""
-"The resource requirement limits for this container.  For example, 'cpu=200m,"
-"memory=512Mi'.  Note that server side components may assign limits depending "
-"on the server configuration, such as limit ranges."
+"这个容器的资源需求限制。 例如，\"cpu=200m,内存=512Mi\"。 请注意，服务器端的组"
```
Comment (tengqm) [inline staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:701]:
```suggestion
"这个容器的资源需求限制。例如，\"cpu=200m,内存=512Mi\"。请注意，服务器端的组"
```

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 7]
```diff
@@ -727,34 +698,33 @@ msgid ""
...
-"The resource requirement requests for this container.  For example, "
-"'cpu=100m,memory=256Mi'.  Note that server side components may assign "
-"requests depending on the server configuration, such as limit ranges."
+"The resource requirement requests for this container.  For example, 'cpu=100m,"
+"memory=256Mi'.  Note that server side components may assign requests "
+"depending on the server configuration, such as limit ranges."
 msgstr ""
-"资源为 container 请求 requests . 例如, 'cpu=100m,memory=256Mi'. 注意服务端组"
-"件也许会赋予 requests, 这决定于服务器端配置, 比如 limit ranges."
+"这个容器的资源需求请求。 例如，\"cpu=200m,内存=512Mi\"。 请注意，服务器端的组"
```
Comment (tengqm) [inline staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:711]:
```suggestion
"这个容器的资源需求请求。例如，\"cpu=200m,内存=512Mi\"。请注意，服务器端的组"
```

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 8]
```diff
@@ -727,34 +698,33 @@ msgid ""
...
+"件可能会根据服务器的配置来分配限制，例如限制范围。"
 
 # https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/create_secret.go#L87
 #: staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret.go:155
 msgid "The type of secret to create"
-msgstr "创建 secret 类型资源"
+msgstr "要创建的 secret 类型"
```
Comment (tengqm) [inline staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:717]:
```suggestion
msgstr "要创建的 Secret 类型"
```

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 9]
```diff
@@ -727,34 +698,33 @@ msgid ""
...
 #: staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_undo.go:87
 msgid "Undo a previous rollout"
-msgstr "撤销上一次的 rollout"
+msgstr "撤销上一次的上线"
 
 # https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/set/set_resources.go#L101
 #: staging/src/k8s.io/kubectl/pkg/cmd/set/set_resources.go:116
 msgid "Update resource requests/limits on objects with pod templates"
-msgstr "在对象的 pod templates 上更新资源的 requests/limits"
+msgstr "使用 pod 模板更新对象的资源请求/限制"
```
Comment (tengqm) [inline staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:727]:
```suggestion
msgstr "使用 Pod 模板更新对象的资源请求/限制"
```

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 10]
```diff
@@ -763,45 +733,44 @@ msgstr "更新一个资源的注解"
 # https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/label.go#L109
 #: staging/src/k8s.io/kubectl/pkg/cmd/label/label.go:133
 msgid "Update the labels on a resource"
-msgstr "更新在这个资源上的 labels"
+msgstr "更新一个资源的标签"
```
Comment (tengqm) [inline staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:736]:
```suggestion
msgstr "更新某资源上的标签"
```

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 11]
```diff
@@ -763,45 +733,44 @@ msgstr "更新一个资源的注解"
...
 #: staging/src/k8s.io/kubectl/pkg/cmd/taint/taint.go:109
 msgid "Update the taints on one or more nodes"
-msgstr "更新一个或者多个 node 上的 taints"
+msgstr "更新一个或者多个节点上的污点"
 
 # https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/create_secret.go#L155
 #: staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret_docker.go:150
 msgid "Username for Docker registry authentication"
-msgstr "Username 为 Docker registry authentication"
+msgstr "Docker registry 身份验证的用户名"
```
Comment (tengqm) [inline staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:746]:
```suggestion
msgstr "用于 Docker 镜像库身份验证的用户名"
```

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 12]
```diff
@@ -763,45 +733,44 @@ msgstr "更新一个资源的注解"
...
 
 # https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/clusterinfo_dump.go#L45
 #: staging/src/k8s.io/kubectl/pkg/cmd/clusterinfo/clusterinfo_dump.go:85
 msgid ""
 "Where to output the files.  If empty or '-' uses stdout, otherwise creates a "
 "directory hierarchy in that directory"
 msgstr ""
-"输出到 files.  如果是 empty or '-' 使用 stdout, 否则创建一个 目录层级在那个目"
-"录"
+"在哪里输出文件。 如果为空或“-”使用标准输出，否则在该目录中创建目录层次结构"
```
Comment (tengqm) [inline staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:759]:
```suggestion
"在哪里输出文件。如果为空或 “-” 则使用标准输出，否则在该目录中创建目录层次结构"
```

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 13]
```diff
@@ -862,8 +831,8 @@ msgstr "kubectl 控制 Kubernetes cluster 管理"
 #~ "password=DOCKER_PASSWORD --docker-email=DOCKER_EMAIL"
 #~ msgstr ""
 #~ "\n"
-#~ "\t\t  # 如果你还没有一个 .dockercfg 文件, 你可以直接使用下面的命令创建一"
-#~ "个 dockercfg 的 secret:\n"
+#~ "\t\t  # 如果你还没有一个 .dockercfg 文件, 你可以直接使用下面的命令创建一个 "
```
Comment (tengqm) [inline staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:834]:
```suggestion
#~ "\t\t  # 如果你还没有 .dockercfg 文件, 你可以直接使用下面的命令创建一个 "
```

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 14]
```diff
@@ -862,8 +831,8 @@ msgstr "kubectl 控制 Kubernetes cluster 管理"
...
+#~ "dockercfg 的 secret:\n"
```
Comment (tengqm) [inline staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:835]:
```suggestion
#~ "dockercfg 类型的 Secret：\n"
```

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 15]
```diff
@@ -1402,8 +1371,8 @@ msgstr "kubectl 控制 Kubernetes cluster 管理"
 #~ "\t\t# 使用 JSON 格式化输出显示一个单独的 pod.\n"
 #~ "\t\tkubectl get -o json pod web-pod-13je7\n"
 #~ "\n"
-#~ "\t\t# 显示一个被 \"pod.yaml\" 中的 type 和 name 标识的 pod 并使用 JSON 格"
-#~ "式化输出.\n"
+#~ "\t\t# 显示一个被 \"pod.yaml\" 中的 type 和 name 标识的 pod 并使用 JSON 格式"
```
Comment (tengqm) [inline staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:1374]: pod 都应拼写为 Pod

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 16]
```diff
@@ -900,8 +869,8 @@ msgstr "kubectl 控制 Kubernetes cluster 管理"
 #~ "\t\tkubectl apply --prune -f manifest.yaml -l app=nginx\n"
 #~ "\n"
 #~ "\t\t# 应用 manifest.yaml 的配置并删除所有不在这个文件中的 configmaps.\n"
```
[the remark this answers is not in the document]
Comment (hwdef) [reply staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:871]: 这里是不是应该为`ConfigMaps。`

[the remark this answers is not in the document]
Comment (chenrui333) [reply staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:871]: 应该是 `ConfigMap`s

File: staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po (hunk 15 above)
Comment (hwdef) [reply staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:1374]: 这些资源在英文原文中是小写的，如果在中文里是大写的，就不一致了，会不会不太好看

Comment (tengqm) [reply staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:1374]: 中文里面夹带英文时，首字母不大写很难看。k/website 的中文翻译中，我们约定：除非字段名确实是首字母小写，保留不变，其他单词都用首字母大写。

Comment (hwdef) [reply staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po:1374]: 好的，那我就按照k/website的约定修改下。
另外有句话不太通顺，我在slack中咨询您了。

**Comment by hwdef** [comment]
@tengqm 
I have modified all places according to your suggestion, please review again.
Thanks!

**Comment by hwdef** [comment]
@deads2k @soltysh 
Please take a look. Thanks.

**Review by soltysh — APPROVED** [review]
I trust the folks who lgtm the PR, since I don't know Chinese :sweat_smile:

**Pull request merged**